extension/README.md

```diff
@@ -2,12 +2,11 @@
 
 The extension does two things:
 
-1) Synchronizes metadata from the browser to the shared Arcs system.
+1) Synchronizes metadata from the browser to your Arc.
 1) Provides an entry point to Arcs.
 
-The first point is worth re-emphasizing. **The extension sends metadata from
-pages you visit to a public firebase.** We've limited the extent of data
-shared (only Product and Event entities are sent).
+Currently (and temporarily) synchronization happens via a **public** firebase.
```

Comment:
Add something on what we plan to do post-temporarily.

Reply:
I think the new design no longer does that. We just inject the pages locally in an arc and run the speculator. Only if the user accepts a suggestion using the data, data will be saved.

Reply:
Good point - I added 'Eventually the sync will be to a private store (perhaps firebase).' as the next line. Do you think that's enough or should I add more?

Reply:
Also it's true, the new design doesn't sync in the same way. I want to be sure before I change the README that it's working as designed.